readme.md

```diff
@@ -64,12 +64,82 @@ money out of it yourself.
 
 To update your own instance, follow the instructions below.
 
+## Running with Docker
+
+You can use [Docker](https://www.docker.com) and
+[docker-compose](https://docs.docker.com/compose/) to pull or build
+and run a Monica image, complete with a self-contained MySQL database.
+This has the nice properties that you don't have to install lots of
+software directly onto your system, and you can be up and running
+quickly with a known working environment.
+
+Before you start, run `cp .env.example .env` and open `.env` in an
```

Comment:
To run locally, you would clone then run the following. However, you should also be able to just pull `monicahq/monicahq` and run it basically like `docker run -e DB_HOST=localhost...<other env> -p 80:80 monicahq/monicahq -d`.